manifests/init.pp

```diff
@@ -89,6 +89,13 @@
     features        => [],
   }
 
+  # check selinux
+  $_selinux = if fact('os.selinux.enabled') and $facts['os']['selinux']['enabled'] and $icinga2::globals::selinux_package_name {
```

Comment:
I think we can simplify it a bit. `fact()` will return the fact value, if the fact exists, or undef:
```suggestion
  $_selinux = if fact('os.selinux.enabled') == true and $icinga2::globals::selinux_package_name {
```

Reply:
I did it the same way as in the icinga2 module. And the unit test on FreeBSD and Windows said it's required.